endpoint-root/endpoint-core/src/main/java/org/divy/common/bo/endpoint/BaseBOEndpoint.java

```diff
@@ -61,9 +61,7 @@ protected E doDelete(I id) {
 
         List<E> boList = manager.list();
 
-        List<E> resultList = new ArrayList<>();
-
-        resultList.addAll(boList);
+        List<E> resultList = new ArrayList<>(boList);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "resultList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)